README.md

```diff
@@ -1 +1,160 @@
-# budget-app
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [BUDGET APP] <a name="about-project"></a>
+
+**[Budget app]** is an application designed for users to track their expenditures.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- <a href="https://www.ruby-lang.org/es/">ROR</a>
+- <a href="https://www.postgresql.org/">Postgresql</a>
+- <a href="https://rspec.info/">Rspec</a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Users can view their transaction and Categories]**
+- **[Users can create a transaction]**
+- **[Users can create a category.]**
+- **[Limit access to web app resources based on authorization rules.]**
+
+- Live Demo [Link](https://budget-app-racha.onrender.com/)
```

Comment:
- [ ] Kindly ensure your app is well deployed. It shows an error when I click the link.

  ![image](https://github.com/rachaouldbabaali/budget-app/assets/54848363/d5cc64f7-9d2f-4a14-8ab1-07706e0e1ca7)